src/server.py

```diff
@@ -34,22 +38,23 @@ def server():
                 message_complete = True
 
         try:
-            parse_request(recv_message.decode('utf-8'))
+            client_uri = parse_request(recv_message.decode('utf-8'))
         except TypeError as msg:
             reply = response_error(405, str(msg))
         except ValueError as msg:
             reply = response_error(400, str(msg))
         else:
-            reply = response_ok()
-
+            resolved = resolve_uri(client_uri)
+            reply = response_ok(resolved)
         conn.sendall(reply)
+
         conn.close()
     server_socket.close()
 
 
-def response_ok():
+def response_ok(body):
     """Return a http response ok message."""
-    my_message = Response(200, 'You did it correctly! YOU ROCK!')
+    my_message = Response(200, body)
```

Comment:
What happens when you build a Response object using the binary data from an image, which is a bytestring?